website/packages/docs/src/pages/media-queries-and-other-at-rules.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 section: 50-Guides
+name: Media queries and other at-rules
```

Comment:
fix capitalisation of the page in the sidebar

![Screenshot 2024-07-11 at 11 29 09](https://github.com/atlassian-labs/compiled/assets/2908767/6f9630e4-ec24-4710-8480-ef53acf18fff)